src/pytorch_lightning/plugins/precision/apex_amp.py

```diff
@@ -55,35 +54,26 @@ def dispatch(self, trainer: "pl.Trainer") -> None:
             self._connected = True
         return super().dispatch(trainer)
 
-    def backward(
-        self,
-        model: "pl.LightningModule",
-        closure_loss: Tensor,
-        optimizer: Optional[Optimizer],
-        optimizer_idx: Optional[int],
-        *args: Any,
-        **kwargs: Any,
-    ) -> None:
+    def backward(self, tensor: Tensor, model: "pl.LightningModule", *args: Any, **kwargs: Any) -> None:
         """Run before precision plugin executes backward.
 
         Args:
+            tensor: the loss value obtained from the closure
             model: the model to be optimized
-            closure_loss: the loss value obtained from the closure
-            optimizer: current optimizer being used. ``None`` if using manual optimization
-            optimizer_idx: the index of the current optimizer. ``None`` if using manual optimization
         """
+        optimizer, optimizer_idx, *args = args
         opt = optimizer or model.trainer.optimizers
-        with amp.scale_loss(closure_loss, opt) as closure_loss:
-            super().backward(model, closure_loss, optimizer, optimizer_idx, *args, **kwargs)
+        with amp.scale_loss(tensor, opt) as tensor:
+            super().backward(tensor, model, optimizer, optimizer_idx, *args, **kwargs)
 
     def optimizer_step(
         self,
-        model: Optional[Union["pl.LightningModule", Module]],
         optimizer: Optimizer,
-        optimizer_idx: int,
-        closure: Callable[[], Any],
+        model: Optional["pl.LightningModule"] = None,
         **kwargs: Any,
     ) -> Any:
+        optimizer_idx = kwargs.pop("optimizer_idx")
+        closure = kwargs.pop("closure")
```

Comment:
I don't get the reason for doing this. How is this different to having them explicitly defined in the signature?

Reply:
Ah okay. This is just to avoid the `override` mypy complaint